index.html

```diff
@@ -513,29 +513,140 @@ <h3>Assert Claim</h3>
       </dl>
     </section>
     <section>
-      <h3>Verify Claim</h3>
+      <h3>Verify Credential</h3>
       <dl class="dl-horizontal">
         <dt>Requirement</dt>
         <dd>
-It MUST be possible for a <a>verifier</a> to verify that the credential is an
-authentic statement of an <a>issuer's</a> <a>claims</a> about the
-<a>subject</a>. The verifying <a>entity</a> must have the capability to
-connect the issuer’s identity to its credential identifier and the
-<a>subject's</a> identity to their identifier as indicated in the credential.
-The issuer’s verification information, such as its public key, must be
-discoverable from the credential record and verifiably linked to the issuer.
-It MUST be possible to do this in an automated fashion.
+          It MUST be possible for a <a>verifier</a> to verify that the
+          credential is an authentic statement of an <a>issuer's</a>
+          <a>claims</a> about the
+          <a>subject</a>. Verifiers must have the capability to
```

Comment:
presuming this is normative section
```suggestion
          <a>subject</a>. Verifiers MUST have the capability to
```